dev-requirements.txt

```diff
@@ -18,6 +18,7 @@ httpx~=0.24.0
 deepdiff~=6.5
 hypothesis[numpy]~=6.87
 pytest-rerunfailures~=13.0
+snowflake-connector-python~=3.7
```

Comment:
Please move this to the system tests section, it's not a part of the general testing deps 🙂